gwlfe/CalcCnErosRunoffSed.py

```diff
@@ -27,6 +27,9 @@ def CalcCN(z, i, Y, j):
     z.UncontrolledQ = 0
     z.RetentionEff = 0
 
+    if not hasattr(z, 'AdjUrbanQTotal'):
+        z.AdjUrbanQTotal = 0
```

Comment:
Consider moving this default value here: https://github.com/WikiWatershed/gwlf-e/blob/develop/gwlfe/datamodel.py#L19